crates/hotshot/src/lib.rs

```diff
@@ -308,25 +308,63 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> SystemContext<TYPES, I> {
         Ok(inner)
     }
 
-    /// "Starts" consensus by sending a `QcFormed` event
+    /// "Starts" consensus by sending a `QcFormed`, `ViewChange`, and `ValidatedStateUpdated` events
     ///
     /// # Panics
     /// Panics if sending genesis fails
     pub async fn start_consensus(&self) {
+        #[cfg(feature = "dependncy-tasks")]
+        error!("HotShot is running with the dependency tasks feature enabled!!");
         debug!("Starting Consensus");
         let consensus = self.consensus.read().await;
+
+        #[allow(clippy::panic)]
         self.internal_event_stream
             .0
             .broadcast_direct(Arc::new(HotShotEvent::ViewChange(self.start_view)))
             .await
-            .expect("Genesis Broadcast failed");
+            .unwrap_or_else(|_| {
+                panic!(
+                    "Genesis Broadcast failed; event = ViewChange({:?})",
+                    self.start_view
+                )
+            });
+        #[cfg(feature = "dependency-tasks")]
```

Comment:
cc @bfish713

Reply:
Yeah this looks reasonable to me, we may have to consider the ability to restart without the state for start view

Reply:
Edited this to only insert the data if it's present for the start view.